HousingSearchApi.Tests/V1/E2ETests/Fixtures/TenureFixture.cs

```diff
@@ -0,0 +1,74 @@
+using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Net.Http;
+using System.Threading;
+using AutoFixture;
+using Elasticsearch.Net;
+using HousingSearchApi.V1.Gateways.Models;
+using Nest;
+
+namespace HousingSearchApi.Tests.V1.E2ETests.Fixtures
+{
+    public class TenureFixture : BaseFixture
+    {
+        public List<QueryablePerson> Persons { get; private set; }
+        private const string INDEX = "tenures";
+        public static string[] Alphabet = { "aa", "bb", "cc", "dd", "ee", "vv", "ww", "xx", "yy", "zz" };
+
+        public TenureFixture(IElasticClient elasticClient, HttpClient httpClient) : base(elasticClient, httpClient)
+        {
+            WaitForESInstance();
+        }
+
+        public void GivenATenureIndexExists()
+        {
+            ElasticSearchClient.Indices.Delete(Indices.Index(INDEX));
+
+            if (!ElasticSearchClient.Indices.Exists(Indices.Index(INDEX)).Exists)
+            {
+                var tenureSettingsDoc = File.ReadAllTextAsync("./data/elasticsearch/tenureIndex.json").Result;
+                ElasticSearchClient.LowLevel.Indices.CreateAsync<BytesResponse>(INDEX, tenureSettingsDoc)
+                    .ConfigureAwait(true);
+
+                var tenures = CreateTenureData();
+                var awaitable = ElasticSearchClient.IndexManyAsync(tenures, INDEX).ConfigureAwait(true);
+
+                while (!awaitable.GetAwaiter().IsCompleted)
+                {
+
```

Comment:
Timeout?

Reply:
I tried to make it work like so, and it still failed. I have -no- clue what's with ES indexing and waiting on it...